### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by lopata54** [Chia-Network/chia-blockchain#14046] [Support]
Hello. help please. I installed your wallet and can't find a solution to the problem. After opening the wallet , there is an infinite configuration . What am I doing wrong?
![2022-12-02_21-52-16](https://user-images.githubusercontent.com/119700380/205368945-9a3fb2e1-2307-45f9-97aa-0c7ddca9e695.png)

**Comment by lopata54** [comment]
how long can I wait for an answer here?

**Comment by lopata54** [comment]
yes

ср, 7 дек. 2022 г. в 15:42, leokryp ***@***.***>:

**Comment by wallentx** [comment]
Are you able to perhaps provide us with some logs?
On Windows, your logs directory can be found at `C:\Users\<Your-User>\.chia\mainnet\log`.
You can also set how verbose your log level is by editing your config at `C:\Users\<Your-User>\.chia\mainnet\config\config.yaml`, and searching for the entry `log_level:`, and changing from the default `INFO` to `DEBUG`.

**Comment by lopata54** [comment]
in the mainnet folder I have only 1 folder

чт, 8 дек. 2022 г. в 12:40, William Allen ***@***.***>:

**Comment by lopata54** [comment]
Did I write something wrong? why did you disappear?

чт, 8 дек. 2022 г. в 12:40, William Allen ***@***.***>:

**Comment by MumfMeisterT** [comment]
Which version are you running? 
You may have hidden files turned off. Try checking those to show and you should find your folders.

**Comment by lopata54** [comment]
of course, hidden files are included in the display. Version 1.6.1. please help me. for a month of correspondence, the question did not budge.

**Comment by mor** [comment]
How old is your computer? It may not be able to sync with the chia-blockchain if it isn't pretty fast, with plenty of RAM (at least 16gb). Reboot it and make sure no other programs are running when you try again. If it still doesn't work, try uninstalling v1.6.1 and installing the v1.6.0 release instead.

**Comment by lopata54** [comment]
everything is fine with the computer. there is enough RAM. I removed version 1.6.1 and installed 1.6.0 and the exact same situation. no folders appeared.

**Reply by leokryp** [reply]
Sorry friend, I accidentally deleted my question. what operating system are you using?

win7 no support
w8 have bug
w8.1 ok
etc... ok
